gdspy/library.py

```diff
@@ -1252,9 +1252,9 @@ def area(self, by_spec=False):
             else:
                 return self.ref_cell.area() * self.magnification ** 2
 
-    def get_polygons(self, by_spec=False, depth=None):
+    def _transform_polygons(self, polygons, by_spec=False):
         """
-        Return the list of polygons created by this reference.
+        Transforms a set of polygons based on the CellReference's applied transformations.
 
         Parameters
         ----------
```

Comment:
Missing documentation of `polygons`. It should be similar to the output of `get_polygons`.
The argument `by_spec` should be removed. It is implicit by checking whether `isinstance(polygons, dict)`.

Reply:
fixed